windows/container.go

```diff
@@ -174,6 +174,11 @@ func (c *container) Pid() uint32 {
 	return c.ctr.Pid()
 }
 
+func (c *container) Ps(ctx context.Context) ([]uint32, error) {
+	// TODO (ehazlett): support finding all processes in windows container
```

Comment:
Ah, missed the above comment. Really, you can only rely on PIDs returned through that HCS call. Especially for Hyper-V containers, where there is no correlation between host PIDs and container PIDs. Further, the API shouldn't allow sending 'signals' (use of quotes as there really aren't signals on Windows) to any process in the container - just those that have been launched through the (equivalent of in the docker world) run/exec commands - not container system services